### PR TITLE
COIL TIER is now listed on a new line in recipeviewers

### PR DESCRIPTION
**Pull request by Ghostipedia**
## What
This PR moves the Coil Tier to a separate line in recipe viewers, allowing pack/mod devs to have more space for coil naming
Before : 
image
After :
image

### CODE REVIEW

File: src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java [hunk 1]
```diff
@@ -456,11 +456,21 @@ public class GTRecipeTypes {
                 ICoilType requiredCoil = ICoilType.getMinRequiredType(temp);
 
                 if (requiredCoil == null || requiredCoil.getMaterial() == null) {
-                    return LocalizationUtils.format("gtceu.recipe.temperature", temp);
+                    return LocalizationUtils.format("gtceu.recipe.temperature");
                 } else {
-                    return LocalizationUtils.format("gtceu.recipe.temperature_and_coil", temp, I18n.get(requiredCoil.getMaterial().getUnlocalizedName()));
+                    return LocalizationUtils.format("gtceu.recipe.temperature", temp);
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java]: uhhh, just do `return LocalizationUtils.format("gtceu.recipe.temperature", temp);` without the if statement

File: src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java [hunk 2]
```diff
@@ -456,11 +456,21 @@ public class GTRecipeTypes {
...
                 }
             })
+            .addDataInfo(data -> {
+                int temp = data.getInt("ebf_temp");
+                ICoilType requiredCoil = ICoilType.getMinRequiredType(temp);
+
+                if (requiredCoil == null || requiredCoil.getMaterial() == null) {
+                return LocalizationUtils.format("gtceu.recipe.coil.tier");
+                } else {
+                return LocalizationUtils.format("gtceu.recipe.coil.tier", I18n.get(requiredCoil.getMaterial().getUnlocalizedName()));
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java]: just don't add the tier line at all if there's no material for it?

**Review by screret — CHANGES_REQUESTED** [review]

File: src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java [hunk 3]
```diff
@@ -451,16 +451,19 @@ public class GTRecipeTypes {
...
-                if (requiredCoil == null || requiredCoil.getMaterial() == null) {
-                    return LocalizationUtils.format("gtceu.recipe.temperature", temp);
-                } else {
-                    return LocalizationUtils.format("gtceu.recipe.temperature_and_coil", temp, I18n.get(requiredCoil.getMaterial().getUnlocalizedName()));
+                if (requiredCoil != null || requiredCoil.getMaterial() != null) {
+                return LocalizationUtils.format("gtceu.recipe.coil.tier", I18n.get(requiredCoil.getMaterial().getUnlocalizedName()));
                 }
-            })
+                return "";
+        })
```
Comment (screret) [inline src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java]: fix formatting?

**Review by screret — CHANGES_REQUESTED** [review]

File: src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java (hunk 3 above)
Comment (Ghostipedia) [reply src/main/java/com/gregtechceu/gtceu/common/data/GTRecipeTypes.java]: sorted

**Review by screret — APPROVED** [review]

**Pull request merged**